service/flags.go

```diff
@@ -64,6 +82,12 @@ func flags() *flag.FlagSet {
 	flagSet.Var(featuregate.FlagValue{}, featureGatesFlag,
 		"Comma-delimited list of feature gate identifiers. Prefix with '-' to disable the feature. '+' or no prefix will enable the feature.")
 
+	flagSet.Var(
+		new(boolValue),
+		watchConfigFlag,
+		"Enable config watch for configuration sources.",
+	)
```

Comment:
flagSet.BoolVar(
		new(bool),
		watchConfigFlag,
		"Enable config watch for configuration sources.",
	)